music21/musicxml/xmlToM21.py

```diff
@@ -3881,11 +3881,18 @@ def xmlTechnicalToArticulation(self, mxObj):
             if tag in ('heel', 'toe'):
                 if mxObj.get('substitution') is not None:
                     tech.substitution = xmlObjects.yesNoToBoolean(mxObj.get('substitution'))
+            if tag == 'bend':
+                tech.bendAlter = interval.Interval(int(mxObj.find('bend-alter').text))
```

Comment:
this has too many potentials for crashing the processing.  What if there is no bend-alter or no bend-alter.text?  check that before typing to assign.

(Better to call out into a separate sub-method.  This method was designed just for one-to-two lines per technical

Reply:
sub-method and bend-alter check in https://github.com/cuthbertLab/music21/pull/1580/commits/2e1ea432934e58cd877346be9b3b3d030c7a9a51